### ISSUE TITLE
Autocomplete text overlapping

### ISSUE THREAD

**Issue by techeverri** [Microsoft/vscode#6274]
- VSCode Version: 1.1.0
- OS Version: Windows 10

Steps to Reproduce:

1. Open **User Settings** file `settings.json`
2. Suppose you are going to add a new setting
3. Press `ctrl` + `space` and scroll to `editor.lineHeight`

![text-overlapping-vscode](https://cloud.githubusercontent.com/assets/3720424/15168974/1e9acb76-16fc-11e6-96bf-48f2ad3ff187.gif)

**Comment by isidorn** [comment]
I can repro this but only on win machine and only one element in suggest box gets broken for me. And only after scrolling a lot.
@egamma opinion if this is important to tackle before Joao returns. My impression is no, since does not always happen and user can get out of this state by reopening suggest

**Comment by joaomoreno** [comment]
Duplicate of #3464
